tracee/container_tracer.py

```diff
@@ -1129,17 +1125,43 @@ def print_event(self, cpu, data, size):
         # log.info(json.dumps(events, indent=4))
         # exit()
 
+    # process event
+    def handle_event(self, cpu, data, size):
```

Comment:
`size` is not used, remove.

Reply:
This is a callback function of open_perf_buffer - signature can not be changed

Reply:
This is a callback function - can't change arguments here